content/deploy/install/services.md

```diff
@@ -317,6 +317,35 @@ alpha:
     replicas: 1
 ```
 
+### Enable Audit Subgraph
```

Comment:
If `subgraph` will be a common term going forward, we also need to add the term it to `accept.txt`. https://github.com/libremfg/libremfg.github.io/blob/main/.github/styles/Vocab/docs/accept.txt 

I can do that. My bad, the prose-checker is a completely undocumented CI action that I just added in #46.

Reply:
Yeah, we will need something in "Concepts" for Subgraph / Supergraph